src/main/java/duke/tasks/TaskDDL.java

```diff
@@ -0,0 +1,32 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Intermediate class to define Tasks with a deadline or timing.
+ */
+public class TaskDDL extends Task {
```

Comment:
Perhaps it would be clearer if we changed the class name to something like TaskWithTiming so that it is more easily understandable